include/SDL3/SDL_pixels.h

```diff
@@ -733,35 +733,26 @@ typedef struct SDL_Palette
 } SDL_Palette;
 
 /**
- * Details about the format of a pixel.
- *
- * Generally this is used with SDL_Surface, and covers many possible
- * configurations, including paletted data and various bit patterns.
+ * Details about the format of a packed RGB or RGBA pixel.
  *
  * \since This struct is available since SDL 3.0.0.
  */
-typedef struct SDL_PixelFormat
+typedef struct SDL_PackedPixelDetails
 {
-    SDL_PixelFormatEnum format;
-    SDL_Palette *palette;
-    Uint8 bits_per_pixel;
-    Uint8 bytes_per_pixel;
```

Comment:
I'm surprised we didn't want to keep one of these just as a convenience. But I guess there's a macro that pulls it right from the enum value, right?